docs/src/main/sphinx/connector/kafka.rst

```diff
@@ -446,6 +446,20 @@ used to resolve the subject name via the topic name. Note that a case
 insensitive match must be done, as identifiers cannot contain upper case
 characters.
 
+.. _kafka-sql-support:
+
+SQL support
+-----------
+
+The connector provides read and write access to data and metadata in Trino
+tables populated by Kafka topics. In addition to the :ref:`globally available <sql-globally-available>`
+and :ref:`read operation <sql-read-operations>` statements, the connector
+supports the following features:
+
+* :doc:`/sql/insert`, encoded to a specified data format. See also
+  :ref:`kafka-sql-inserts`.
```

Comment:
Similar to this for reads you might also want to link to https://trino.io/docs/current/connector/kafka.html#row-decoding.

Reply:
I would argue that it would be misleading to link to that section here, because the connector supports read operations [out of the box](https://trino.io/docs/current/connector/kafka-tutorial.html#step-4-basic-data-querying) whereas INSERT explicitly requires row encoding.

Reply:
Decoding sounds more like it belongs in a type mapping section?

Reply:
There are 3 scenarios:

- Using a schema registry. In this case reads would work magically.
- Reading messages into the predefined columns (key, message, timestamp, partition etc.). This is what works if you don't have table definitions. Note that the message part doesn't get read back as multiple columns - it's just a big old string.
- Table definitions. This is needed if you are not using a registry and want to read the messages broken out into columns.

Reply:
The issue is the layout is a bit confusing.

Ideally it should be:

- Schema Registry
  - ...
- No schema registry
  - Kafka Inserts
    - Row encoding
  - Kafka Reads
    - Row decoding

Because when you have the registry you don't need to follow any other step - neither of Row encoding or Row decoding.

Reply:
We can improve this as a follow-up but we should definitely link to `Row decoding` for the time being here.

Reply:
Sounds good & agree on the need for a restructure. Added a link to Row Decoding for now.